src/mono/mono/mini/mini-llvm.c

```diff
@@ -7713,29 +7713,73 @@ process_bb (EmitContext *ctx, MonoBasicBlock *bb)
 				break;
 			case OP_FMAX:
 			case OP_FMIN: {
-#if defined(TARGET_X86) || defined(TARGET_AMD64)
 				LLVMValueRef args [] = { l, r };
-
+#if defined(TARGET_X86) || defined(TARGET_AMD64)
+				LLVMTypeRef t = LLVMTypeOf (l);
+				unsigned int elems = LLVMGetVectorSize (t);
 				gboolean is_r4 = ins->inst_c1 == MONO_TYPE_R4;
-				if (ins->inst_c0 == OP_FMAX)
-					result = call_intrins (ctx, is_r4 ? INTRINS_SSE_MAXPS : INTRINS_SSE_MAXPD, args, dname);
-				else
-					result = call_intrins (ctx, is_r4 ? INTRINS_SSE_MINPS : INTRINS_SSE_MINPD, args, dname);
+				int iid = -1;
+				if (ins->inst_c0 == OP_FMAX) {
+					if (elems == 1)
+						iid = is_r4 ? INTRINS_SSE_MAXSS : INTRINS_SSE_MAXSD;
+					else
+						iid = is_r4 ? INTRINS_SSE_MAXPS : INTRINS_SSE_MAXPD;
+				} else {
+					if (elems == 1)
+						iid = is_r4 ? INTRINS_SSE_MINSS : INTRINS_SSE_MINSD;
+					else
+						iid = is_r4 ? INTRINS_SSE_MINPS : INTRINS_SSE_MINPD;
+				}
+				result = call_intrins (ctx, iid, args, dname);
+#elif defined(TARGET_ARM64)
+				int instc0_arm64 = ins->inst_c0 == OP_FMAX ? INTRINS_AARCH64_ADV_SIMD_FMAX : INTRINS_AARCH64_ADV_SIMD_FMIN;
```

Comment:
This could just set 'iid'.

Reply:
Will fix this in a follow-up PR. I'd like to get this merged before the Preview2 snap.

Reply:
Fixed via #65445